Lib/defcon/objects/font.py

```diff
@@ -1052,9 +1055,8 @@ def _glyphRenamedNotificationCallback(self, notification):
             if oldName in layer:
                 oldStillExists = True
                 break
-        if not oldStillExists:
-            self.updateGlyphOrder(removedGlyph=oldName)
-        self.updateGlyphOrder(addedGlyph=newName)
+        removedGlyph = oldName if not oldStillExists else None
```

Comment:
so we don't really remove it if the old name is in any other layer. possibly related https://github.com/trufont/trufont/issues/386#issuecomment-287545776

is there a test for this?